playwright/tests/homePageElements.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { Given, Then } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given("I am on the DRB home page", async function (this: CustomWorld) {
+  return await this.page.goto("https://digital-research-books-beta.nypl.org/");
+});
+
+Then("I see the site name", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("//h1[contains(text(), 'Digital Research Books')]")
+  ).toBeVisible();
+});
+
+Then("I see the Home breadcrumb", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("//ol/li/a/span[contains(text(), 'Home')]")
+  ).toBeVisible();
+});
+
+Then("I see the Research breadcrumb", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("//ol/li[2]/a/span[contains(text(), 'Research')]")
+  ).toBeVisible();
+});
+
+Then(
+  "I see the Digital Research Books Beta breadcrumb",
+  async function (this: CustomWorld) {
+    await expect(
+      this.page.locator(
+        "//ol/li[3]/span/span[contains(text(), 'Digital Research Books Beta')]"
```

Comment:
To remove nested selectors here, let's change to something like `      this.page.locator("li", {hasText: "Digital Research Books Beta"})`

Reply:
done.